docs/en/operations/table_engines/custom_compression_codec.md

```diff
@@ -0,0 +1,44 @@
+
+# Column compression codecs
+
+Besides default data compression, defined in [server settings](../server_settings/settings.md#compression), per-column specification is also available.
+
+Supported compression algorithms:
+
+- `NONE` - no compression for data applied
+- `LZ4`
+- `LZ4HC(level)` - (level) - LZ4 compression algorithm with defined level.
+Possible `level` range: [1,12]. Greater values stands for better compression and higher CPU usage. Recommended value range: [4,9].
```

Comment:
https://github.com/lz4/lz4/blob/c10863b98e1503af90616ae99725ecd120265dfb/lib/lz4hc.h#L46-L50 [3-12], in other cases 9.